CMakeLists.txt

```diff
@@ -316,11 +337,9 @@ endif()
 #-----------------------------------------------------------------------------
 # Testing
 #-----------------------------------------------------------------------------
-if(CMAKE_PROJECT_NAME STREQUAL HERMES)
+if(NOT HERMES_EXTERNALLY_CONFIGURED AND BUILD_TESTING)
+  enable_testing()
```

Comment:
From [the documentation](https://cmake.org/cmake/help/v3.10/command/add_test.html#command:add_test):
> CMake will generate tests only if the enable_testing() command has been invoked. **The CTest module invokes the command automatically when the BUILD_TESTING option is ON.**